### DISCUSSION TITLE
对于刚上手的新人

### DISCUSSION THREAD

**Discussion by ghost** [mamoe/mirai#2759] [Ideas]
对于“万事开头难”，这个咱们但都是知道的，mirai可以出一个官方视频，用于如何去设置以及Java和Kotlin，如何去部署以及启动mirai机器人，虽然文档也是个好的方法，但是，不如手把手教出来的好，对于新手确实是很难上手的，真心给你们提个建议。

**Comment by zhaodice** [comment]
mirai正在以 `114514 Line/ 1 Decade` 的速度迭代中,即使是出了教程，也很快有一些api或者写法过时，最后还得回到看文档

**Reply by ghost** [reply]
谢谢，我是想根据mirai来写一个基于mirai自己的一个框架，我也很有幸加入到mirai开发这个项目中

**Comment by zhaodice** [comment]
如果是单纯的run的话，其实更建议用衍生项目MCL
https://github.com/iTXTech/mcl-installer
